feature/detailnews/src/androidTest/java/kr/co/cotton/feature/detailnews/ExampleInstrumentedTest.kt

```diff
@@ -0,0 +1,24 @@
+package kr.co.cotton.feature.detailnews
+
+import androidx.test.platform.app.InstrumentationRegistry
+import androidx.test.ext.junit.runners.AndroidJUnit4
+
+import org.junit.Test
+import org.junit.runner.RunWith
+
+import org.junit.Assert.*
```

Comment:
🚫 **[ktlint]** <no-wildcard-imports> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Wildcard import (cannot be auto-corrected)